### PR TITLE
Added redirects for MCAL subproject 

### PR DESCRIPTION
**Pull request by jrvosse**
- added Jacco as contact point in readme
- added mcal redirects to triple store hosted by triply

### CODE REVIEW

File: odissei/ns/mcal/.htaccess [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Options +FollowSymLinks
+RewriteEngine on
+
+RewriteRule ^/ns/mcal/(.*)$ http://api.odissei.triply.cc/redirect/https://w3id.org/odissei/ns/mcal/$1 [R=307,L]
```
Comment (davidlehn) [inline odissei/ns/mcal/.htaccess:5]: RewriteRules are relative to the current directory.  I'm guessing you wanted this:
```suggestion
RewriteRule ^(.*)$ http://api.odissei.triply.cc/redirect/https://w3id.org/odissei/ns/mcal/$1 [R=307,L]
```

**Comment by dgarijo** [comment]
Ping @ritamargherita

**Comment by jrvosse** [comment]
Addressed comments by @davidlehn

**Comment by dgarijo** [comment]
Still, since you are adding more maintainers, we need the ok from @ritamargherita, the previous sole maintainer of the w3id.

**Comment by ritamargherita** [comment]
@dgarijo yes I approve the changes made by @jrvosse

**Pull request merged**